dataclients/kubernetes/ingressv1.go

```diff
@@ -105,6 +105,7 @@ func convertPathRuleV1(
 		r := &eskip.Route{
 			Id:          routeID(ns, name, host, prule.Path, svcName),
 			Backend:     eps[0],
+			BackendType: eskip.NetworkBackend,
```

Comment:
I guess this is just for clarity as zero value for `BackendType` is `NetworkBackend`